src/index.ts

```diff
@@ -75,6 +76,28 @@ export type FhevmInstance = {
   } | null;
 };
 
+/**
+ * @param {string}  relayerUrl - Relayer's URL.
+ * @param {number} chainId - FHEVM host chain id.
+ * @param {string=} [publicKeyId] - Optional public key id.
+ * @param {Eip1193Provider | string} [network] - Optional network.
+ */
+export const createInstanceFromRelayer = async (
```

Comment:
To be honest, I'd be opinionated here and not create a special `createInstanceFromRelayer` which shares half of the parameter with `createInstance`

I like the endpoint you've made, but I would go 100% in this direction.

`createInstance('sepoliaEthereum')` (note: sepolia Base exists)

Then, you have a mapping in the code
```
sepoliaEthereum: {
  chainId: 11155111,
  relayerUrl: "https://relayer.sepolia.zama.ai"
  ...
} 
```

Reply:
Then you have 2 usecases to handle:
- **People who wants to have a specific config (like us on staging, for the moment)**. It could be solved if the relayer just return what we want.

- **Public key caching**
There are two strategies:
  - relayer-sdk fetch keys and store them in the localstorage if the lib is run in a browser. It's totally transparent for the dapp developer. However, it's impossible to do something similar in node.
  - relayer-sdk fetch keys and expose a method to export the material and let the dapp developer handle the storage. It's the current strategy.
  
So far, we wanted the library "agnostic" and not too opinionated to be compatible with any partners using FHEVM or a fork. It's not the case anymore, so I'd push for the first solution.